### PR TITLE
feat:(ckan) add sdmx transfer api version

### PR DESCRIPTION
**Pull request by Locutus66**
- added value for sdmx transfer api version for deployment
- increment chart value

### CODE REVIEW

File: stable/ckan/values.yaml [hunk 1]
```diff
@@ -38,6 +38,9 @@ extensions:
     azureTenantIdUtil: "yep.. lemme just paste that azureTenantIdUtil here..."
     azureKeyVaultUrlUtil: "yep.. lemme just paste that azureKeyVaultUrlUtil here..."
 
+  sdmx:
```
Comment (blairdrummond) [inline stable/ckan/values.yaml:41]: You're missing the "enabled" flag

**Review by blairdrummond — CHANGES_REQUESTED** [review]

File: stable/ckan/values.yaml (hunk 1 above)
Comment (Locutus66) [reply stable/ckan/values.yaml:41]: Ah indeed, good catch! - added.

**Review by blairdrummond — APPROVED** [review]
LGTM!

**Comment by blairdrummond** [comment]
Looks like there are merge conflicts

**Pull request merged**